### PR TITLE
Made changes to readme.md

### PR DESCRIPTION
**Pull request by wenweil**
rewrote the description for -i option for clarity.

rewrote the cookies section so it is easier to read.

fixed a small wording mistake

### CODE REVIEW

**Comment by wenweil** [comment]
There seemed to be a problem with your automated integration test. All I changed in this commit is the readme file, which does not affect the code

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ $ annie -i https://youtu.be/Gnbch2osEeo
      # download with: annie -f default "URL"
 ```
 
-Use `annie -f format "URL"` to download a specific format.
+Use `annie -f format "URL"` to download a specific format listed in the output of ' -i ' option.
```
Comment (iawia002) [inline README.md]: of `-i` ...

File: README.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ $ annie https://youtu.be/Gnbch2osEeo
 >
 > `$ annie 'https://...'`
 
-The `-i` option displays all available formats, information without downloading.
+The `-i` option displays all information of video in the supplied link without downloading.
```
Comment (iawia002) [inline README.md]: `all information` seems not suitable, how about `all available quality and formats`

I think this looks simpler:

```
The `-i` option displays all available quality and formats without downloading.
```

**Comment by iawia002** [comment]
> There seemed to be a problem with your automated integration test. All I changed in this commit is the readme file, which does not affect the code

Don't worry about CI, next time you can add `[ci skip]` to your commit message, it will skip the CI.

File: README.md [hunk 3]
```diff
@@ -160,21 +160,18 @@ A temporary `.download` file is kept in the output directory. If `annie` is run
 
 Cookies can be provided to `annie` with the `-c` option if they are required for accessing the video.
 
-**Note:**
+Cookies can be a string or a text file.
 
-* cookies must match the following format:
```
Comment (iawia002) [inline README.md]: This format needs to be retained.

I prefer this:

**Note: cookies must match the following format:**

```
name=value; name2=value2; ...
```

Cookies can be a string or a text file.

As a string:
...

**Pull request merged**